starter-code/src/app/contact-list/contact-list.component.html

```diff
@@ -1,18 +1,34 @@
+<div class="container">
+<div *ngFor="let contact of contacts">
+  <h1>{{contact.name}}</h1>
+  <p>{{contact.email}}</p>
+  <p>{{contact.phoneNumber}}</p>
+  <img [src]="contact.image">
+</div>
+
+
+<h2>Add new contact</h2>
+<div class="field">
 <div>
   <label for="contact-name"></label>
-  <input type="text" [(ngModel)]="newContact.name">
+  <input type="text"  [(ngModel)]="newContact.name">
 </div>
 <div>
   <label for="contact-email"></label>
   <input type="text" [(ngModel)]="newContact.email">
 </div>
 <div>
   <label for="contact-phone"></label>
-  <input type="text" [(ngModel)]="newContact.phoneNumber">
+  <input type="text"  [(ngModel)]="newContact.phoneNumber">
 </div>
 <div>
   <label for="contact-image"></label>
   <input type="text" [(ngModel)]="newContact.image">
 </div>
 
-<button (click)="addContact">Submit New Contact</button>
+</div>
+
+<div class="field submit">
+<button type="submit" (click)="addContact(newContactInput)">Submit New Contact</button>
```

Comment:
As we saw, we do not need the newContactInput here. The component already knows about newContact